src/content/docs/release-notes/agent-release-notes/ruby-release-notes/ruby-agent-800.mdx

```diff
@@ -3,7 +3,7 @@ subject: Ruby agent
 releaseDate: '2021-09-22'
 version: 8.0.0
 downloadLink: 'https://rubygems.org/downloads/newrelic_rpm-8.0.0.gem'
-features: ["Distributed tracing enabled by default", "Cross Application Tracing is deprecated and disabled by default", "Update configuration default value for span_events.max_samples_stored from 1000 to 2000", "add_method_tracer now uses Module prepending instead of alias_method chaining", "Add support for ARM64/Graviton2 platform using Ruby 3.0.2+"]
+features: ["Distributed tracing enabled by default", "Cross Application Tracing is deprecated and disabled by default", "Update configuration default value for span_events.max_samples_stored from 1000 to 2000", "add_method_tracer now uses Module prepending instead of alias_method chaining", "Add support for ARM64/Graviton2 platform using Ruby 3.0.2+", "Remove previously deprecated method External.start_segment", "Remove support for Excon versions below 0.19.0, Mongo versions below 2.1, and Merb (all versions)"]
```

Comment:
```suggestion
features: ["Distributed tracing enabled by default", "Cross Application Tracing is deprecated and disabled by default", "Update configuration default value for span_events.max_samples_stored from 1000 to 2000", "add_method_tracer now uses Module prepending instead of alias_method chaining", "Add support for ARM64/Graviton2 platform using Ruby 3.0.2+", "Remove previously deprecated method External.start_segment", "Remove support for Excon versions below 0.19.0," "Remove support for Mongo versions below 2.1", "Remove support for Merb"]
```